praw/models/inbox.py

```diff
@@ -189,7 +190,7 @@ def message(self, message_id: str) -> praw.models.Message:
 
         """
         listing = self._reddit.get(API_PATH["message"].format(id=message_id))
-        messages = {message.fullname: message for message in [listing[0]] + listing[0].replies}
+        messages = {message.fullname: message for message in [listing[0], *listing[0].replies]}
```

Comment:
Is there a performance implication here?

Reply:
It seems like there is -- this was an auto fix: https://docs.astral.sh/ruff/rules/collection-literal-concatenation/